tests/Unit/Elliptic/Initialization/Test_Source.cpp

```diff
@@ -76,7 +76,6 @@ SPECTRE_TEST_CASE("Unit.Elliptic.Initialization.Source",
     ActionTesting::MockRuntimeSystem<Metavariables<1>> runner{
         {AnalyticSolution<1>{}}, {}};
 
-    MockParallelComponent component{};
```

Comment:
Could you also remove line 61 in this file, as well as `Test_BoundaryConditions.cpp:93`?